pkgdb2/templates/package.html

```diff
@@ -200,8 +200,13 @@
 {% block content %}
 
 <section id="pkg_info">
-  <h1 class="inline" property="doap:name">{{ package.name }}</h1>{% if package.upstream_url
-      %} (<a class="inline" href="{{ package.upstream_url }}">upstream</a>){% endif %}
+  <h1 class="inline">
+      <span class="italic">{{ package.namespace | capitalize}}:</span>
```

Comment:
Replace the ``:`` with ``/``?  We want to encourage people to think of the namespaces as directories everywhere I think (that was what I got out of the devel list discussion).  I.e.:

- rpms/nethack
- rpms/cockpit

and

- dockerfiles/cockpit

In this section of the template, can we ensure there is no space inbetween the ``/`` and the text on either side, too?